test/focusable-behavior.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>d2l-focusable-behavior tests</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+		<script src="../../web-component-tester/browser.js"></script>
+		<link rel="import" href="../../polymer/polymer.html">
+		<link rel="import" href="../d2l-focusable-behavior.html">
+		<link rel="import" href="../d2l-dom-focus.html">
+	</head>
+	<body>
+
+		<dom-module id="d2l-focusable-test">
+			<template>
+				<div>
+					<div>
+						<a id="shadow1" href="javascript:void(0);"></a>
+					</div>
+					<div>
+						<slot></slot>
+					</div>
+					<div>
+						<a id="shadow2" class="d2l-focusable" href="javascript:void(0);"></a>
```

Comment:
I added this extra `d2l-focusable` class to the distributed light-DOM element for the purpose of the tests to ensure that the focus would target the focusable element in the shadow-DOM rather than a distributed child that might also happen to have this class name.

Reply:
Note: I should have actually made this comment for the `d2l-focusable` a few lines further down.  ;)